client/Makefile

```diff
@@ -36,19 +36,15 @@ start-frontend:
 e2e:
 	JEST_ENV=${PROD} \
 	CXG_CLIENT_PORT=${CXG_SERVER_PORT} \
-		node node_modules/jest/bin/jest.js \
+		npm run e2e -- \
```

Comment:
also uses `npm run e2e` in makefile script now 👌

Reply:
Nice!